api/src/main/java/uk/gov/dhsc/htbhf/claimant/communications/DetermineEntitlementNotificationHandler.java

```diff
@@ -20,13 +21,17 @@
     private final MessageQueueClient messageQueueClient;
 
     public void sendClaimNoLongerEligibleEmail(Claim claim) {
```

Comment:
Probably best if we don't even call this method if email is not present. In which case you won't have to change any logic here